src/open_inwoner/components/templates/components/Form/FileInput.html

```diff
@@ -5,7 +5,7 @@
 <div class="form__control file-input" aria-live="polite">
     {% render_card direction="vertical" %}
         {% icon icon="upload" icon_position="before" outlined=True %}
-        <input class="file-input__input" id="{{ field.auto_id }}" name="file" type="file"{% if field.field.required %} required{% endif %}{% if multiple %} multiple{% endif %} data-max-size="{{ openzaak_config.max_upload_size }}">
+        <input class="file-input__input" id="{{ field.auto_id }}" name="file" type="file"{% if field.field.required %} required{% endif %}{% if multiple %} multiple{% endif %} data-max-size="{{ openzaak_config.max_upload_size }}" data-file-types="{{ openzaak_config.allowed_file_extensions }}">
```

Comment:
I'm not sure if this `FileInput` is used anywhere other than the document upload form, but since it looks like a generic template it seems wrong to directly use `openzaak_config` here.

I'd prefer it if we pass something like `allowed_file_extensions=openzaak_config.allowed_file_extensions` here: https://github.com/maykinmedia/open-inwoner/blob/e8471d865eb52e26543b9c378d8299ec77952b18/src/open_inwoner/templates/pages/cases/document_form.html#L6 and then directly use `allowed_file_extensions` in the `FileInput` template

Reply:
Ok @stevenbal then I think I should also do this for the max-file-size.